spec/controllers/groups_controller_spec.rb

```diff
@@ -7,7 +7,7 @@
     it 'create a group when supplied valid params' do
       post :create, group: {
         group_name: 'Group A' }
-      expect(response).to be_success
+      expect(response).to redirect_to action: :index
```

Comment:
hmm...not really, the code before the changes fails this test as it expects a 200 rather than a 302.

![screen shot 2016-07-03 at 10 09 09 pm](https://cloud.githubusercontent.com/assets/3384784/16549252/fc8b143e-416a-11e6-97e5-e65da1bf5cde.png)